templates/etc_default_grub.j2

```diff
@@ -0,0 +1,13 @@
+{{ ansible_managed | comment }}
+{{ "system_role:bootloader" | comment(prefix="", postfix="") }}
+GRUB_TIMEOUT={{ bootloader_timeout }}
+GRUB_DISTRIBUTOR="$(sed 's, release .*$,,g' /etc/system-release)"
+GRUB_DEFAULT=saved
+GRUB_DISABLE_SUBMENU=true
+GRUB_TERMINAL_OUTPUT="console"
+GRUB_CMDLINE_LINUX={{ grub_cmdline_linux }}
```

Comment:
```suggestion
GRUB_CMDLINE_LINUX="{{ grub_cmdline_linux }}"
```
Does this need to be double quoted?

Reply:
> The output of `Get bootloader args of the default kernel` task returns this quoted. On my Fedora:
> 
> ```
> $ sudo grubby --info=DEFAULT | grep -oP 'args=\K(.*)'
> "$tuned_params ro rootflags=subvol=root rd.luks.uuid=luks-9da1fdf5-14ac-49fd-a388-8b1ee48f3df1 rhgb quiet"
> ```

ah - so the double quotes are actually part of the `grub_cmdline_linux` string?  ok